### ISSUE TITLE
[Question] I trained the llava-1.5-13B model, but when evaluating, the inference answer is always empty and the speed is very slow.

### ISSUE THREAD

**Issue by ykzqjyhhh** [haotian-liu/LLaVA#1778]
### Question

![20241122185229](https://github.com/user-attachments/assets/4318b4f7-31c1-45ce-8912-c8cfd8d9f9a2)
The training loss graph looks like this, which is normal.
The training configuration is the same as pretrain.sh

**Comment by CAOANJIA** [comment]
Same problem. My finetuned model always outputs empty string and the speed is very slow. 

Have you solved it?

**Comment by yinyuanzhang** [comment]
Same problem.  How did you solve it?

**Comment by CAOANJIA** [comment]
> solve

maybe you need to look at the forward in vision encoder, notice data type (list or not)

**Comment by MangoLibn** [comment]
Thank u for your insight!! but I didn't understand how to fix it, because i follow the original code strictly.

Can u talk it in detail ? Thank u Very much!!

**Comment by MangoLibn** [comment]
> > solve
> 
> maybe you need to look at the forward in vision encoder, notice data type (list or not)

Thank u for your insight!! but I didn't understand how to fix it, because i follow the original code strictly.

Can u talk it in detail ? Thank u Very much!!

**Comment by MassEast** [comment]
I am experiencing the same problem; model only outputs "</s>".

**Comment by MassEast** [comment]
For me, what helped was to stay close to [model_vqa.py](https://github.com/haotian-liu/LLaVA/blob/main/llava/eval/model_vqa.py) as explained in the [Evaluation](https://github.com/haotian-liu/LLaVA/blob/main/docs/Evaluation.md) section. Especially, do not forget to use a conversation template for your prompt!
